app/build.gradle

```diff
@@ -110,6 +110,12 @@ dependencies {
     implementation 'com.android.support:design:26.0.1'
     implementation 'pl.droidsonroids.gif:android-gif-drawable:1.2.17'
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'com.facebook.fresco:fresco:2.0.0'
+    implementation 'com.facebook.fresco:animated-gif:2.0.0'
```

Comment:
I don't this this libraries are necessary. The only I think you need is `implementation 'com.facebook.fresco:fresco:2.0.0'`